lang/src/test/kotlin/org/partiql/lang/ExprValueStrictEqualsTest.kt

```diff
@@ -0,0 +1,152 @@
+package org.partiql.lang
```

Comment:
Actually, I don't know if the packaging of `ExprValueStrictEquals.kt` & `ExprValueStrictEqualsTest.kt` is correct. The former is a tool used in test, and the latter is test to this tool.